.github/workflows/trigger-helm-deploy.yaml

```diff
@@ -0,0 +1,64 @@
+---
+name: trigger-helm-deploy
+on:
+  workflow_dispatch:
+    inputs:
+      circle_context:
+        description: "CI context for aws credentials"
+        default: "sentinel-staging-deploy"
+      kubernetes_cluster:
+        description: "kubernetes cluster"
+        default: "mainnet-us-east-2-dev-eks"
+      aws_region:
+        description: "aws region"
+        default: "us-east-2"
+      namespace:
+        description: "kubernetes namespace"
+        default: "ntwk-nerpanet-dealbot"
+      release:
+        description: "helm release"
+        default: "dealbot-0"
+      chart:
+        description: "the helm chart"
+        default: "filecoin/dealbot"
+      chart_version:
+        description: "the helm chart version"
+        default: "0.0.5"
+      helm_append:
+        description: "append this to the end of the helm command (i.e to add more -set's)"
+      override_repository:
+        description: "helm chart image.repository"
+      override_tag:
+        description: "helm chart image.tag"
+      slack_channel:
+        description: "the slack channel where notifications are sent"
+        default: "netops-circleci-test"
+      circle_branch:
+        description: "circleci branch (used for testing new lotus-infra deploy code)"
+        default: master
+
+jobs:
+  trigger-circleci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: trigger
+        uses: promiseofcake/circleci-trigger-action@v1
```

Comment:
should we pin this to a specific commit?

Reply:
The version tag should be sufficient.

Reply:
see e.g. https://github.com/ipld/.github/pull/18 - i believe we tried to have a policy of pinning 3rd party actions by hash

Reply:
Right, you don't want to give that user a way to run arbitrary code in our CI, especially if there's secrets involved :)